libs/domain/search/facet-value-navigation/facet-value-navigation.styles.ts

```diff
@@ -25,7 +25,7 @@ export const facetValueNavigationStyles = css`
 
   slot:not([name]) + oryx-button {
     width: fit-content;
-    padding-block-end: 10px;
+    padding-block-start: 10px;
```

Comment:
Change of ui can be a breaking change :(

Reply:
True. But this is a small bug, hence the change without warning.